libvirt/tests/src/passthrough/pci/libvirt_pci_passthrough_eeh.py

```
@@ -0,0 +1,401 @@
+        6. check device hotplugged or not
+        7. check STORAGE device inside guest
+        8. Get location code for the pci device
+        9. mount file (/dev/nvme*) on /mnt
+        9. Trigger EEH on pci device
+        10. Check EEH recovery on the device
+        11. check device availability inside guest
+    """
+    # get the params from params
+    vm_name = params.get("main_vm")
+    vm = env.get_vm(vm_name)
+    device_name = params.get("libvirt_pci_net_dev_name", "ENTER_YOUR.DEV.NAME")
```

Ruff (0.13.1)

Local variable `device_name` is assigned to but never used

Remove assignment to unused variable `device_name`

(F841)